api/lib/backends/UHT-Contacts/FetchRecord.js

```diff
@@ -80,7 +78,7 @@ const processCustomer = results => {
     .filter(x => x);
 
   customer.tenancies = tenancies.map(tenancy => {
-    if (tenancy.endDate.includes('1900-01-01')) {
+    if (tenancy.endDate.getYear === '1990') {
```

Comment:
Should this be 1900?

Reply:
Think so. The default for dates seems to be 1990-01-01 which means it's not valid and should really be null, so checking if the year is set to 1990 will almost certainly grab those non valid dates. Me thinks.

Reply:
I think you've got an extra 9 in there - 1900 != 19_9_0

Reply:
haha, so I do. Thanks for catching that!

Reply:
:-)